docs/pages/button.js

```diff
@@ -27,88 +32,75 @@ card(
         required: false,
         defaultValue: null,
         description: [
-          'Supply a short, descriptive label for screen-readers to replace button texts that do not provide sufficient context about the button component behavior. Texts like `Click Here,` `Follow,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the button text.',
-          'Use IconButton if you only need a button with an icon and no text.',
-          'Accessibility: It populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of `text`.',
+          'Label to provide more context around Button’s function or purpose. See the [Accessibility guidelines](/accessibility) to learn more.',
         ],
-        href: 'accessibilityLabel',
+        href: 'Accessibility-props:-controls-expanded-and-popup',
```

Comment:
the new docs don't support links on the prop names. You can remove all this href sections.

Reply:
Thanks! Done!